Form/form_us.xml

```diff
@@ -2403,7 +2403,7 @@
                    <_attribute>Months unemployed during census year</_attribute>
                </column>
                <column>
-                   <_attribute>(Day of the Enumerator’s visit) sick or temporarily disabled</_attribute>
+                   <_attribute>(Day of the Enumerator's visit) sick or temporarily disabled</_attribute>
```

Comment:
Do you really want to do this?  Changing an attribute will cause problems for existing users of the form.

Reply:
The changes being requested will not effect any data that has already been entered. After sebschub confirms his issue has been resolved the new file can be released. The corrected file is at https://www.dropbox.com/personal/Public/Gramps%204.2?preview=form_us.xml

Reply:
The address to the corrected file should be https://www.dropbox.com/s/8u5hwjuu17agxga/form_us.xml?dl=0

Reply:
@sebschub Can you confirm please?